plugins/heroku/test/promoteStagingToProduction.test.ts

```diff
@@ -3,9 +3,8 @@ import { promoteStagingToProduction } from '../src/promoteStagingToProduction'
 import heroku from '../src/herokuClient'
 import { gtg } from '../src/gtg'
 import winston, { Logger } from 'winston'
-import { setAppConfigVars } from '../src/setConfigVars'
 
-const logger = (winston as unknown) as Logger
+const logger = winston as unknown as Logger
```

Comment:
**question**: i'm sure this keeps flipflopping, what's causing that?

Reply:
looks like it's just the heroku tests that still have parentheses in this casting at this point, i expect we're just slowly applying prettier throughout the source as we make changes. (i still need to fix this test logging, winston doesn't like you just using the root logger without any transports configured and will print loads of complaints to stderr as soon as you switch off `--silent`)